packages/google_atlas/lib/src/google_atlas.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
+import 'dart:ui';
 
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
```

Comment:
Why is material needed? I think all the widgets used here are part `package:flutter/widgets.dart`, right?